### PR TITLE
Performance boost

### PR DESCRIPTION
**Pull request by sciabarracom**
Improved performaces up to 1100-1200 requests / second removing all the goroutines and the detection t ahah child process is dead.

### CODE REVIEW

File: examples/benchmark/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+all: golang bash
+
+test.lua:
+	echo 'wrk.method = "POST"'>test.lua
+	echo "wrk.body = '{\"value\":{\"name\":\"Mike\"}}'">>test.lua
+	echo 'wrk.headers["Content-Type"] = "application/json"'>>test.lua
+
+golang: test.lua
+	docker run -d --name under-test --rm -p 8080:8080 $(IMG)
+	invoke init main.go
```
Comment (Juice10) [inline examples/benchmark/Makefile]: Would be nice to include the `invoke` file/command in this PR

File: examples/benchmark/init.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (dubee) [inline examples/benchmark/init.sh]: @sciabarra, looks like Travis failed as a license header is missing from this file.

File: examples/benchmark/Makefile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	wrk -t1 -c1 -stest.lua http://localhost:8080/run
+	docker kill under-test
+
+bash: test.lua
+	docker run -d --name under-test --rm -p 8080:8080 $(IMG2)
+	bash init.sh main.sh
+	wrk -t1 -c1 -stest.lua http://localhost:8080/run
+	docker kill under-test
+
+.PHONY: all golang bash
```
Comment (csantanapr) [inline examples/benchmark/Makefile]: nice addition for benchmarking both images 🎉

**Comment by csantanapr** [comment]
Thanks @michele-sciabarra for this great improvements. 🎉 
Thanks @Juice10 and @dubee for reviewing and merging. I just got to this now. 🤗

**Pull request merged**